src/exercises/2-Design_Tokens/theme.js

```diff
@@ -1,6 +1,9 @@
 const theme = {
 	colors: {
-		myfavoritecolor: '#f39037',
+		background: '#fff',
```

Comment:
Wasn't adding these colors part of the exercise here?

Reply:
It was but I think it's not worth exploring how to add entries to the object but rather give a pointer and let people explore. We might define the task a bit differently (change tokens to match a design). I think in this case it was again the fact that theme-ui defines required fields in the theme. Thoughts?

Reply:
Do they define required colors? Worth mentioning it's somewhere and link to their documentation.